src/overlay_mic_test.c

```diff
@@ -0,0 +1,868 @@
+    BgTemplate bgTemplate;
+    u8 unk1C;
+    u8 unk1D;
+    u8 unk1E;
+    u8 unk1F;
+} ov62_021E6728[] = {
+    {
+     .bgTemplate = {
+            .x          = 0,
+            .y          = 0,
+            .bufferSize = 0x800,
+            .baseTile   = 0,
+            .size       = 1,
+            .colorMode  = 0,
+            .screenBase = 0x1F,
+            .charBase   = 0,
+            .bgExtPltt  = 0,
+            .priority   = 0,
+            .areaOver   = 0,
+            .dummy      = 0,
+            .mosaic     = 0 },
+     .unk1C = 0,
+     .unk1D = 0,
+     .unk1E = 0,
+     .unk1F = 0,
+     },
+    {
+     .bgTemplate = { .x = 0, .y = 0, .bufferSize = 0x800, .baseTile = 0, .size = 1, .colorMode = 0, .screenBase = 0x1E, .charBase = 1, .bgExtPltt = 0, .priority = 1, .areaOver = 0, .dummy = 0, .mosaic = 0 },
+     .unk1C      = 1,
+     .unk1D      = 0,
+     .unk1E      = 0,
+     .unk1F      = 0,
+     },
+    {
+     .bgTemplate = { .x = 0, .y = 0, .bufferSize = 0x800, .baseTile = 0, .size = 1, .colorMode = 0, .screenBase = 0x1D, .charBase = 2, .bgExtPltt = 0, .priority = 2, .areaOver = 0, .dummy = 0, .mosaic = 0 },
+     .unk1C      = 2,
+     .unk1D      = 0,
+     .unk1E      = 0,
+     .unk1F      = 0,
+     },
+    {
+     .bgTemplate = { .x = 0, .y = 0, .bufferSize = 0x800, .baseTile = 0, .size = 1, .colorMode = 0, .screenBase = 0x1F, .charBase = 0, .bgExtPltt = 0, .priority = 0, .areaOver = 0, .dummy = 0, .mosaic = 0 },
+     .unk1C      = 4,
+     .unk1D      = 0,
+     .unk1E      = 0,
+     .unk1F      = 0,
+     },
+    {
+     .bgTemplate = { .x = 0, .y = 0, .bufferSize = 0x800, .baseTile = 0, .size = 1, .colorMode = 0, .screenBase = 0x1E, .charBase = 1, .bgExtPltt = 0, .priority = 1, .areaOver = 0, .dummy = 0, .mosaic = 0 },
+     .unk1C      = 5,
+     .unk1D      = 0,
+     .unk1E      = 0,
+     .unk1F      = 0,
+     },
+};
+
+static const UnkTemplate_0200D748 ov62_021E67C8[5] = {
+    { .x = 0x40, .y = 0x60, .z = 0, .animation = 0, .spritePriority = 0, .pal = 0, .vram = NNS_G2D_VRAM_TYPE_2DMAIN, .resIdList = { 0xA01, 0xA03, 0xA02, 0xA04, 0, 0 }, .bgPriority = 0, .vramTransfer = 0 },
+    { .x = 0xC0, .y = 0x60, .z = 0, .animation = 0, .spritePriority = 0, .pal = 0, .vram = NNS_G2D_VRAM_TYPE_2DMAIN, .resIdList = { 0xA01, 0xA03, 0xA02, 0xA04, 0, 0 }, .bgPriority = 0, .vramTransfer = 0 },
+    { .x = 0x80, .y = 0x60, .z = 0, .animation = 0, .spritePriority = 0, .pal = 0, .vram = NNS_G2D_VRAM_TYPE_2DMAIN, .resIdList = { 0xB01, 0xB03, 0xB02, 0xB04, 0, 0 }, .bgPriority = 0, .vramTransfer = 0 },
+    { .x = 0xD8, .y = 0xB0, .z = 0, .animation = 0, .spritePriority = 0, .pal = 0, .vram = NNS_G2D_VRAM_TYPE_2DSUB,  .resIdList = { 0xD01, 0xD03, 0xD02, 0xD04, 1, 0 }, .bgPriority = 0, .vramTransfer = 0 },
+    { .x = 0,    .y = 0,    .z = 0, .animation = 0, .spritePriority = 0, .pal = 0, .vram = NNS_G2D_VRAM_TYPE_2DMAIN, .resIdList = { 0xC01, 0xC03, 0xC02, 0xC04, 0, 0 }, .bgPriority = 0, .vramTransfer = 0 },
```

Comment:
if using designated initializers, each field should be on its own line. exception is unions (i.e. TouchscreenHitbox, below)